backend/src/repository/Boards/Boards.model.js

```diff
@@ -0,0 +1,28 @@
+import mongoose from 'mongoose';
+
+const BoardSchema = new mongoose.Schema({
```

Comment:
nazwę pliku na: MongoBoardsModel.js, hmm?
Abyśmy nie pomylili, jaki to model jest? (i raczej bez tej kropki pomiędzy)